### DISCUSSION TITLE
What is the suggested size for the cover image? All my images appear as cut in height

### DISCUSSION THREAD

**Discussion by andreagrandi** [CaiJimmy/hugo-theme-stack#951] [General]
Hi everyone, while I'm migrating my content to Hugo, I've noticed that images that I chose as cover image are always cut on top when shown in the home page, for example:

<img width="807" alt="Screenshot 2024-02-20 at 18 50 10" src="https://github.com/CaiJimmy/hugo-theme-stack/assets/636391/8579aee8-6d77-4c8e-a227-874fb4d64f8c">

vs the original size:

<img width="1129" alt="Screenshot 2024-02-20 at 18 50 36" src="https://github.com/CaiJimmy/hugo-theme-stack/assets/636391/2943b40a-9b58-4ea9-88b2-217cb09aa2fb">

I wish the theme allowed me to fit for height or width, but instead it just cut the height and the image looks bad.

Should I use a specific size for this image?

Is there a way to fix this? Thanks

**Comment by CaiJimmy** [comment]
Hi, sorry for the late reply,

> I wish the theme allowed me to fit for height or width, but instead it just cut the height and the image looks bad.

This comment explains what is happening underneath: https://github.com/CaiJimmy/hugo-theme-stack/issues/581#issuecomment-1147373812

You can override that CSS file, and remove the `height` property. This should make the image scale proportionally.
